### PR TITLE
Data overhaul

### PR DESCRIPTION
**Pull request by zsteinmetz**
I just opened a draft PR so that we can discuss on the code. This refers to #45.

### CODE REVIEW

**Comment by zsteinmetz** [comment]
Some comments:

- Concerning 481dc4f, could you please add the complete rsconnect folder to the .gitignore file in shiny/
- We'll need to decide whether we want to have the code in R/create_metadata.R in the package or just as a helper function for the Shiny part. I'm I right that it won't be of much use to the R package user?

**Comment by wincowgerDEV** [comment]
> * Concerning [481dc4f](https://github.com/wincowgerDEV/OpenSpecy/commit/481dc4fe2f7c54b80eb923fa90d6c379c77e6cc3), could you please add the complete rsconnect folder to the .gitignore file in shiny/
Done, pushing now.

**Comment by wincowgerDEV** [comment]
> * We'll need to decide whether we want to have the code in R/create_metadata.R ([bd72b26](https://github.com/wincowgerDEV/OpenSpecy/commit/bd72b261d807eac705bcc2c2fcfcad6204c6e267)) inside the package or just as a helper function for the Shiny part (could be sourced). I'm I right that it won't be of much use to the R package user?

I will loop back to this at the end, I haven't decided whether it should be a function at all, it may be a single use item in the shiny app. Still figuring out how to configure it.

**Comment by wincowgerDEV** [comment]
Abandoning ship on the cookies, I have spent the past two days trying to make them work and all they do is give me inconsistencies. Figured out a different workaround to log the user data. I may also try to hash the user identity.  We have so few users right now, I think it could work.

**Comment by wincowgerDEV** [comment]
I think at some point we want to add in an optional login feature. That will be the best way to keep track of users and compartmentalize their data. I see this exercise as getting us closer to that goal. I also see that as a lower priority right now for development since folks aren't really requesting that.

**Comment by wincowgerDEV** [comment]
@zsteinmetz, is it alright with you if I change the way that the metadata is getting uploaded when the share is selected with a new function? I am having a hard time developing the share_spec function to account for every type of data sharing that we have and I am not sure if we actually want it to handle all of those use cases.

**Comment by wincowgerDEV** [comment]
I got the browser fingerprinting working now and logging. I think that will do for now until we figure out a login option for advanced users.

File: R/adj_intens.R [hunk 1]
```diff
@@ -82,7 +84,7 @@ adj_intens.data.frame <- function(x, ...) {
 adj_intens.default <- function(x, y, type = "none", make_rel = TRUE,
                                ...) {
   yadj <- switch(type,
-                 "reflectance" = (1 - adj_neg(y))^2 / (2 * adj_neg(y)),
```
Comment (wincowgerDEV) [inline R/adj_intens.R:85]: #52 corrected here and updated the documentation.

**Review by wincowgerDEV — APPROVED** [review]
I have reviewed all changes and tested the app for stability. Now passing my checks.

**Pull request merged**